src/main.rs

```diff
@@ -116,21 +104,42 @@ impl<'a> Mqtt<'a> {
             .iter()
             .enumerate()
         {
-            let Ok(Event::Incoming(Packet::Publish(n))) = notification else { continue };
-
             let client = self
                 .mqtt_client
                 .client
                 .as_mut()
                 .expect("MqttClient::client not set");
 
-            // Check whether incoming topic matches, if so set pin
-            for p in pins.iter() {
-                if n.topic == p.mqtt_topic_set {
-                    p.set_and_publish_state(client, n.payload.to_owned())?;
-                    continue;
+            match notification {
+                Ok(Event::Incoming(Packet::Publish(n))) => {
+                    // Check whether incoming topic matches, if so set pin
+                    for p in pins.iter() {
+                        if n.topic == p.mqtt_topic_set {
+                            p.set_and_publish_state(client, n.payload.to_owned())?;
+                            continue;
```

Comment:
I know it's been there before, but I think this continue can be removed. I was also considering `break`, but there might be a usecase where multiple pins listen on the same topic.

Reply:
Makes sense.  I'm not sure the particular use-case where you'd set multiple pins for a given topic, but if someone specifies it, it seems like we should honor it.